services/blockstorage/adapter/filesystem/file_system_persistence.go

```diff
@@ -38,7 +38,7 @@ const blocksFilename = "blocks"
 func generateBlockStorageMetrics(m metric.Factory) *metrics {
 	return &metrics{
 		sizeOnDisk:          m.NewGauge("BlockStorage.FileSystemSize.Bytes"),
-		indexLastUpdateTime: m.NewGauge("BlockStorage.FileSystemIndex.LastUpdateTime"),
```

Comment:
in my opinion this is not needed you can immediately call the update in this funciton ... for the 1 nano it is 0 is good enough.